libs/sdk-ui-dashboard/src/_staging/dashboard/fluidLayout/facade/interfaces.ts

```diff
@@ -143,7 +143,7 @@ export interface IDashboardLayoutSectionsFacade<TWidget> {
  * this is the place where you can put the new methods to cover it.
  *
  * Note: This class serves mainly for getting data from the layout.
- *       If you are looking for the layout transformations, check {@link IDashboardLayoutBuilder}
+ *       If you are looking for the layout transformations, check IDashboardLayoutBuilder
```

Comment:
I had to remove this link otherwise i must expose many more interfaces related to the builder.